web/ui/www/static/js/whisper.js

```diff
@@ -43,6 +47,42 @@ function notifySuccess (text) {
     notify("success", text)
 }
 
+function isPasswordValid (password, username, email) {
+    var minCharacters = 12;
+    var maxCharacters = 30;
+    var minUnique = 7;
```

Comment:
@abilioesteves should I retrieve these values with a request? Or pass them somehow via template?

Reply:
via template, please

Reply:
I do it via hidden input or there is a better way?